Makefile

```diff
@@ -11,21 +11,27 @@ include $(BP_SDK_DIR)/Makefile.prog
 override TARGET_DIRS := $(BP_SDK_BIN_DIR) $(BP_SDK_LIB_DIR) $(BP_SDK_INCLUDE_DIR) $(BP_SDK_TOUCH_DIR)
 $(TARGET_DIRS):
 	mkdir -p $@
+	
+sdk_lite: cd $(BP_SDK_DIR); curl -L $(git config --get remote.origin.url)/releases/download/$(git describe --tags --abbrev=0)/sdk_lite.tgz | tar -xvz
 
-sdk_lite: | $(TARGET_DIRS)
+sdk: cd $(BP_SDK_DIR); curl -L $(git config --get remote.origin.url)/releases/download/$(git describe --tags --abbrev=0)/sdk.tgz | tar -xvz
+
+prog: cd $(BP_SDK_DIR); curl -L $(git config --get remote.origin.url)/releases/download/$(git describe --tags --abbrev=0)/prog.tgz | tar -xvz
+
+build_sdk_lite: | $(TARGET_DIRS)
 	cd $(BP_SDK_DIR); git submodule update --init --checkout $(SHALLOW_SUB)
 	$(MAKE) dromajo
 
 ## This target makes the sdk tools
-sdk: sdk_lite
+build_sdk: build_sdk_lite
```

Comment:
keep the same names; the download flow can be pull_sdk? download_sdk? something like that